content/manuals/build/buildkit/dockerfile-release-notes.md

```diff
@@ -13,6 +13,35 @@ issues, and bug fixes in [Dockerfile reference](/reference/dockerfile.md).
 
 For usage, see the [Dockerfile frontend syntax](frontend.md) page.
 
+## 1.16.0
+
+{{< release-date date="2025-05-22" >}}
+
+The full release note for this release is available
```

Comment:
```suggestion
The full release notes for this release is available
```

Reply:
Hum yeah we should change this everywhere else then :sweat_smile: https://github.com/search?q=repo%3Adocker%2Fdocs%20%22The%20full%20release%20note%20for%20this%20release%20is%20available%22&type=code